### ISSUE TITLE
Nested .msg file parsing result is .text instead of .json

### ISSUE THREAD

**Issue by BenJoParadise** [mattgwwalker/msg-extractor#63]
In order to get your bug addressed in a timely manner, or at all :smiley:, please fill out the below bug report. Please try to make it as easy as possible for us to understand what is going on. We may close out any bugs or issues without warning that are not complete or coherent.

In the bug template below anything is [square brackets] should be filled out or removed if the item doesn't apply.

Should you encounter an error that has not already been reported, please do the following when reporting it:
**Bug Metadata**
* - [X] Using latest supported version of extract_msg
* Your python version: Python 2.7
* How did you launch extract_msg?
  - [X] My command line or
  - [ ] I used the extract_msg package

**Describe the bug**
When running the package from command line with the `--json` argument, the outer .msg file parsing is outputted in JSON format, but parsing of  any nested .msg files is outputted as text format.

[ If applicable ]
**What code did you use or can we use to reproduce this error?

```
python extract_msg --json message.msg
```
* Use a message file that has an attached .msg file inside.

Is there a message.msg file you want to share to help us reproduce this?
- [X] Uploaded message: [This was crafted for Ben2.msg.zip](https://github.com/mattgwwalker/msg-extractor/files/2672030/This.was.crafted.for.Ben2.msg.zip)
- [ ] Emailed message as an attachment to admins: [Enter Subject Line Here]

**Screenshots**
[Insert any screenshots or debug pictures here]

**Additional context**
Here is the output of running the command on the file i've attached:
```
.
├── 2018-12-12_1247 This was crafted for Ben
│   ├── 2018-12-12_1247 
│   │   ├── Sheleg.png
│   │   └── message.text
│   └── message.json
└── This was crafted for Ben2.msg
```

**Comment by BenJoParadise** [comment]
Solved it!
In `extract_msg/message.py` -> `save` function -> `# Save the attachment`, replace this line:
`
attachmentNames.append(attachment.save(ContentId))
`
with
`
attachmentNames.append(attachment.save(ContentId, toJson))
`

Generally, you'd probably want to pass on all of the parameters (raw, etc...)

**Comment by TheElementalOfCreation** [comment]
Thank you for bringing this to our attention. This will be fixed in version 0.21.0
